libs/designer-ui/src/lib/settings/settingsection/settingTokenField.tsx

```diff
@@ -154,6 +154,7 @@ export const TokenField = ({
           getTokenPicker={getTokenPicker}
           onChange={onValueChange}
           onMenuOpen={onComboboxMenuOpen}
+          multiSelect={editorOptions?.multiSelect}
```

Comment:
You also probably need to add `serialization={editorOptions?.serialization}` here, the dropdown component has it and it looks like you added it inside the combobox but you aren't passing it in